client/my-sites/sites/sites.jsx

```diff
@@ -125,9 +146,16 @@ export const Sites = React.createClass( {
 } );
 
 const selectSite = ( siteId, rawPath ) => ( dispatch, getState ) => {
-	const path = ( rawPath === '/sites' )
-		? '/stats/insights'
-		: rawPath;
+	let path = rawPath.toLowerCase();
+
+	if ( startsWith( path, '/sites' ) ) {
```

Comment:
Why does `startsWith()` make more sense than comparison with `===`?

Reply:
@jeffstieler The path contains the site URL too, so we need starts with or indexof. Lodash seemed cleaner.